### DISCUSSION TITLE
Cleaner way to show one to many / many to one?

### DISCUSSION THREAD

**Discussion by nordcart** [directus/directus#8282] [Feature Requests]
This is how it looks now:
![image](https://user-images.githubusercontent.com/12813484/134647347-7578bc3d-fd6d-48ed-94d3-c400499a10b8.png)

Imagine if there are hundreds of items, the list would be enormus. 

Could the old way to display these work better to show many objects? A scrollable list? 
Could An option be added in the settings?

Could these be sorted by for example date?

**Comment by benhaynes** [comment]
Both interface styles certainly have their benefits/detriments... I agree we should have a table view too, but it's up to @rijkvanzanten if that's an option or separate interface.

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/issues/3889 👍🏻
